### ISSUE TITLE
Problem PS Move 3 (Windows 10 - 1909)

### ISSUE THREAD

**Issue by probeberja** [HipsterSloth/PSMoveSteamVRBridge#112]
Hello,
my controler PS Move 3 is stuck at 2/7  -->  5/7 when i try to connect it anyone able to help?.   

Windows 10 - 1909

Thanks

**Comment by KSPON21** [comment]
Im having the same issue but im also waiting for someone to explain how to fix.

**Comment by Silvyrfir** [comment]
The cause is that after Windows 10 1903 registry handling has changed. This was fixed on September 2019 but no one has yet made a release of PSMoveService (and so of PSMoveSteamVRBridge either) that includes the fix [#634](https://github.com/psmoveservice/PSMoveService/pull/634).

If you are handy you can get the latest version of [PSMoveService](https://github.com/psmoveservice/PSMoveService) and build it yourself and run the config tool as normal. If not there is [a built binary file](https://1drv.ms/u/s!An8ls2qnV1z_n_57p7s2utrsq_x0LQ) of the service by who made the fix ([source](https://github.com/psmoveservice/PSMoveService/issues/631#issuecomment-515784561)).

You will have to unpair the device from windows Bluetooth menu before it works on the fixed version. If it is still trying to get a key instead of creating one remove all subkeys ("folders") under HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\BTHPORT\Parameters\Devices in registry.


### TL;DR
There is an unreleased fix. A version of service with fix can be found [here](https://1drv.ms/u/s!An8ls2qnV1z_n_57p7s2utrsq_x0LQ) ([source](https://github.com/psmoveservice/PSMoveService/issues/631#issuecomment-515784561)).
1. Remove any old Motion Controller pairing from Windows Bluetooth menu. 
2. Run the fixed PSMoveService.
3. Run the config tool and connect.
4. Try pairing your device as before.
If still didn't work remove all folders from HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\BTHPORT\Parameters\Devices with regedit and try again.


Hope this saves hours of headache from the next person.
Cheers!

**Comment by MrChinman** [comment]
Could someone please re-upload the built binary file? It's no longer available.
